src/test/scala/uk/gov/hmrc/agentmtdidentifiers/model/UtrSpec.scala

```diff
@@ -36,11 +47,15 @@ class UtrSpec extends FlatSpec with Matchers {
     Utr.isValid("200000000B") shouldBe false
   }
 
+  it should "be false when it us akk non-digit characters" in {
```

Comment:
True, I am happy to remove if you are happy @sarthak83? Spelling could do with some work too.